source/framework/include/marlin/BookStoreManager.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+// -- marlin headers
+#include <marlin/MarlinBookConfig.h>
+#include <marlin/Logging.h>
+
+namespace marlin {
+  
+  // forward declaration
+  class Application ;
+  class Processor ;
+
+  /**
+   *  @brief  BookStoreManager class
+   */
+  class BookStoreManager {
+  public:
+    using Logger = Logging::Logger ;
+    
+  public:
+    BookStoreManager() { std::cout << "BSM constructed:!";} ;
```

Comment:
Never use cout. the global macro for logging stuff is `streamlog_out(LEVEL)` instead of cout, i.e  `streamlog_out(MESSAGE)`. It requiries to include Logging.h also

Reply:
I will remember, this should be deleted anyway.